Optimize link redirection (#172)
MystenLabs/move-book: reference/unit-testing.md

## Running Unit Tests

> Unit tests for a [Move package](./packages) can be run with the `sui move test` command.

Reviewer: ```suggestion
Use the `sui move test` command to run unit tests for a [Move package](./packages).
```
